config/_base.js

```diff
@@ -68,6 +68,7 @@ Edit at Your Own Risk
 // ------------------------------------
 // Environment
 // ------------------------------------
+// N.B.: globals added here must _also_ be added to .eslintrc
```

Comment:
Um, sorry. abbreviation meaning "Note Well"; https://en.wikipedia.org/wiki/Nota_bene. Kind of extra emphasis. I can take it out if it's too obscure.